channeldb/graph.go

```diff
@@ -66,7 +66,7 @@ var (
 	// Additionally, the items in this bucket also contain the complete
 	// edge information for a channel. The edge information includes the
 	// capacity of the channel, the nodes that made the channel, etc.  This
-	// bucket resides within the edgeBucket above.  Creation of a edge
+	// bucket resides within the edgeBucket above.  Creation of an edge
```

Comment:
I believe this is intentional. It is the same style used on the lines above :-)

Reply:
I see. Since we're already editing this comment, would you mind removing the extra space from this line and the one above? I'm sure it was just a mistake :)